### DISCUSSION TITLE
Introduction of a few features that can turn revolutionary:P

### DISCUSSION THREAD

**Discussion by AstroDema** [stoatchat/discussions#123] [Feature Suggestions]
It is really nice to see Revolt getting better and developing promising features than discord. So, here are a few features I would like to see.

- **Pins:** Just like Discord, this enables you to pin messages in the server.
- **Inbox/Alert Tab:** Another feature available in Discord, Users should be able to see the mentions in a particular tab.
- **Custom Emoji:** A user can create custom emojis and save it n their profile. This could be used anywhere on the platform. User Emojis as they are called can be limited to ten emojis per user. Whereas, Server Custom Emojis can be limited to thirty emojis per server.
- **Animated Emoji Support:** Just like the name, emojis but animated.

That's all for now, will be back.

**Comment by Rexogamer** [comment]
(per [the release tracker](https://github.com/orgs/revoltchat/projects/2), the first and to an extent the third idea are planned)

**Comment by erkinalp** [comment]
Custom message routing (fosscord is also planning this feature in https://github.com/fosscord/fosscord-server/issues/291)?

**Comment by Ashe** [comment]
The shared calendar from guilded is really really nice, although I'd want the fundamental features in first!
